client/src/util/stateManager/annotationsHelpers.ts

```diff
@@ -3,6 +3,7 @@ Helper functions for user-editable annotations state management.
 See also reducers/annotations.js
 */
 
+import type AnnoMatrix from "../../annoMatrix/annoMatrix";
```

Comment:
These helpers expose `isUserAnnotation` and the new writable-column lookups used throughout the autosave flow. If you’re wondering how we decide whether a column should persist, this is the logic.